app/controlplane/pkg/biz/orgmetrics.go

```diff
@@ -149,6 +160,45 @@ func (uc *OrgMetricsUseCase) TopWorkflowsByRunsCount(ctx context.Context, orgID
 	return uc.repo.TopWorkflowsByRunsCount(ctx, orgUUID, numWorkflows, timeWindow)
 }
 
+// GetLastWorkflowStatusByRun returns the last status of each workflow by its last run
+func (uc *OrgMetricsUseCase) GetLastWorkflowStatusByRun(orgName string) ([]*prometheuscollector.WorkflowLastStatusByRunReport, error) {
```

Comment:
We should indicate that this will only return finalized states, in other words, it will skip reporting on workflows that are in progress. Also, we need to add some testing to this.